quick-protobuf/tests/rust_protobuf/common/mesos/v1.rs

```diff
@@ -1188,10 +1188,7 @@ pub struct HTTPCheckInfo<'a> {
 
 impl<'a> MessageRead<'a> for HTTPCheckInfo<'a> {
     fn from_reader(r: &mut BytesReader, bytes: &'a [u8]) -> Result<Self> {
-        let mut msg = HTTPCheckInfo {
-            protocol: mesos::v1::mod_NetworkInfo::Protocol::IPv4,
-            ..Self::default()
-        };
+        let mut msg = Self::default();
```

Comment:
Looks suspicious.

Reply:
Looking at the generated code, I don't think it is functionally any different. The default value for `mesos::v1::mod_NetworkInfo::Protocol` is `IPv4`, so calling default on `HTTPCheckInfo` should use the default for `Protocol`, correct?

Thoughts?